<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="inputFileLabel">
       <property name="text">
        <string>Select input file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="inputFileLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="inputFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="outputFileLabel">
       <property name="text">
        <string>Select output file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="outputFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="outputFileLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>434</width>
       <height>186</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="md5Tab">
      <attribute name="title">
       <string>MD5</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>44</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="md5Llabel">
         <property name="text">
          <string>Writing in output file calculated MD5 hash sum for input file</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>44</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="md5Button">
           <property name="text">
            <string>Hash it</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>md5Llabel</zorder>
      <zorder>verticalSpacer</zorder>
      <zorder>verticalSpacer_2</zorder>
     </widget>
     <widget class="QWidget" name="sha1Tab">
      <attribute name="title">
       <string>SHA1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>44</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="sha1Llabel">
         <property name="text">
          <string>Writing in output file calculated SHA1 hash sum for input file</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>44</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sha1Button">
           <property name="text">
            <string>Hash it</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>sha1Llabel</zorder>
      <zorder>verticalSpacer_3</zorder>
      <zorder>verticalSpacer_4</zorder>
     </widget>
     <widget class="QWidget" name="ripemd160Tab">
      <attribute name="title">
       <string>Ripemd160</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>44</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ripemd160Llabel">
         <property name="text">
          <string>Writing in output file calculated Ripemd160 hash sum for input file</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>44</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ripemd160Button">
           <property name="text">
            <string>Hash it</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>ripemd160Llabel</zorder>
      <zorder>verticalSpacer_5</zorder>
      <zorder>verticalSpacer_6</zorder>
     </widget>
     <widget class="QWidget" name="rsaTab">
      <attribute name="title">
       <string>RSA</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_24">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <layout class="QGridLayout" name="gridLayout_21">
           <item row="0" column="0">
            <widget class="QLabel" name="rsaPublicKeyFileLabel">
             <property name="text">
              <string>Select PublicKeyFILE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="rsaPublicKeyFileLineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="rsaPublicKeyFileButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_22">
           <item row="0" column="0">
            <widget class="QLabel" name="rsaPrivateKeyFileLabel">
             <property name="text">
              <string>Select PrivateKeyFILE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="rsaPrivateKeyFileLineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="rsaPrivateKeyFileButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_23">
           <item row="0" column="0">
            <widget class="QLabel" name="rsaRandomizeFileLabel">
             <property name="text">
              <string>Select RandomizeFILE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="rsaRandomizeFileLineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="rsaRandomizeFileButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_20">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="QLabel" name="rsaModeLabel">
             <property name="text">
              <string>Mode</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="rsaComboBox">
             <item>
              <property name="text">
               <string>Generate</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Encrypt</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dercypt</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="rsaDoItButton">
           <property name="text">
            <string>Do it</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <widget class="QLabel" name="rsaKeySizeLabel">
             <property name="text">
              <string>Key size</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="rsaKeySizeSpinBox">
             <property name="minimum">
              <number>32</number>
             </property>
             <property name="maximum">
              <number>4096</number>
             </property>
             <property name="singleStep">
              <number>8</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_19">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
      <zorder></zorder>
     </widget>
     <widget class="QWidget" name="elgamalTab">
      <attribute name="title">
       <string>Elgamal</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="elgamalModeLabel">
             <property name="text">
              <string>Mode</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="elgamalComboBox">
             <item>
              <property name="text">
               <string>Generate</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Encrypt</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dercypt</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="elgamalDoItButton">
           <property name="text">
            <string>Do it</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="elgamalKeySizeLabel">
             <property name="text">
              <string>Key size</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="elgamalKeySizeSpinBox">
             <property name="minimum">
              <number>32</number>
             </property>
             <property name="maximum">
              <number>4096</number>
             </property>
             <property name="singleStep">
              <number>8</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="elgamalPublicKeyFileLabel">
             <property name="text">
              <string>Select PublicKeyFILE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="elgamalPublicKeyFileLineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="elgamalPublicKeyFileButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="elgamalPrivateKeyFileLabel">
             <property name="text">
              <string>Select PrivateKeyFILE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="elgamalPrivateKeyFileLineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="elgamalPrivateKeyFileButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QLabel" name="elgamalRandomizeFileLabel">
             <property name="text">
              <string>Select RandomizeFILE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="elgamalRandomizeFileLineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="elgamalRandomizeFileButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>327</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>tabWidget</zorder>
  <zorder></zorder>
  <zorder>textBrowser</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
